src/components/Table/Table.test.tsx

```diff
@@ -97,17 +97,23 @@ describe('Table', () => {
       <Table
         columns={[
           {
-            content: 'Title',
+            content: {
+              label: 'Title',
+              key: 'title',
```

Comment:
You can rename the `key` prop to `sortingKey` to indicate its purpose.

Reply:
I was actually contemplating whether or not to use something more generic like `key` or make it more specific, like `sortingKey`. But since it is used only for the sorting functionality and some test ids, I guess it makes sense to name it `sortingKey`. Ty!